pandas/io/json/_json.py

```diff
@@ -850,6 +850,26 @@ def _convert_axes(self):
         obj = self.obj
         assert obj is not None  # for mypy
         for axis_name in obj._AXIS_ORDERS:
+            if (
+                self.orient == "split"
+                and axis_name == "index"
+                and obj._get_axis(axis_name).dtype == "object"
+                and obj.index.inferred_type == "string"
+            ):
+                index_list = list(obj._get_axis(axis_name).values)
```

Comment:
do we have to convert to a list here? We should be able to inspect `values` directly I think